src/graph/KokkosGraph_GraphColorHandle.hpp

```diff
@@ -483,11 +498,12 @@ class GraphColoringHandle
       size_type new_num_edge = 0;
       typedef Kokkos::RangePolicy<HandleExecSpace> my_exec_space;
 
-      if (
+      if ( 0
```

Comment:
What is this if (0) ?

Reply:
Is this a compiler trick ?

Reply:
I think we talked about this one last week, I think what this:

```C++
        if ( 0
#if defined( KOKKOS_ENABLE_CUDA )
        || Kokkos::Impl::is_same<Kokkos::Cuda, ExecutionSpace >::value
#endif
         )
```

is saying is that the stuff in the block should only get executed if `KOKKOS_ENABLE_CUDA` is true -and- if the ExecutionSpace is `Kokkos::Cuda`, otherwise it should not get executed. I didn't write this thing so I'm assuming it's correct.